test/integration_test/test-deploy.sh

```diff
@@ -56,8 +56,16 @@ KUBEVERSION=$(kubectl version -o json | jq ".serverVersion.gitVersion" -r)
 sed -i 's/<kube_version>/'"$KUBEVERSION"'/g' /specs/stork-scheduler.yaml
 sed -i 's/'stork:.*'/'"$image_name"'/g' /specs/stork-deployment.yaml
 
+# For integration test mock times
+kubectl delete cm stork-mock-time  -n kube-system || true
```

Comment:
Have added an extra `|| true` post PR approval to handle cases where the stork-mock-time configmap may not exist. Don't want the script command to fail.